submissions/Temu4/tiny-js-world/index.js

```diff
@@ -0,0 +1,74 @@
+// ======== OBJECTS DEFINITIONS ========
+const man = {
+  species: 'human',
+  gender: 'male',
+  name: 'John',
+  age: 28,
+  hands: 2,
+  legs: 2,
+  hobby: 'fishing',
+  speach: 'Hello, people!'
+};
+
+const woman = {
+  species: 'human',
+  gender: 'female',
+  name: 'Anny',
+  age: 23,
+  hands: 2,
+  legs: 2,
+  hobby: 'cycling',
+  speach: 'Hello, everybody!'
+};
+
+const dog = {
+  species: 'dog',
+  gender: 'male',
+  name: 'Rex',
+  age: 3,
+  hands: 0,
+  legs: 4,
+  hobby: 'diging bones',
+  speach: 'Wohw!'
+};
+
+const cat = {
+  species: 'cat',
+  gender: 'female',
+  name: 'Sida',
+  age: 1.5,
+  hands: 0,
+  legs: 4,
+  hobby: 'sleaping',
+  speach: 'Meow!'
+};
+
+//Create massage from object properties
+const createMassage = obj => {
```

Comment:
Do you really create `massage` from object ? IMO this function must be called `createMessage`.

Reply:
fixed in [this](https://github.com/kottans/frontend-2019-p2p/pull/44/commits/d61d4ad6ed1c790df4a99d35ec1712f511603671) commit.